src/Modules/BuildList.lua

```diff
@@ -141,7 +156,14 @@ end
 
 function listMode:BuildList()
 	wipeTable(self.list)
-	local handle = NewFileSearch(main.buildPath..self.subPath.."*.xml")
+	-- local handle = NewFileSearch(main.buildPath..self.subPath.."*.xml")
+	local filterList = main.filterBuildList or ""
+	local handle = nil
+	if filterList ~= "" and filterList ~= nil then
+		handle = NewFileSearch(main.buildPath..self.subPath..filterList..".xml")
```

Comment:
This only matches on an exact filename match, not making it very useful for filtering.

Reply:
I don't understand this point. In the EditControl you can type things like "*BLE*" and this, filter all filenames that contains this. Please, could you please let me know what regarding you? An example?

Reply:
![image](https://user-images.githubusercontent.com/1209372/114950803-75a43980-9e19-11eb-9b5e-4f549bab0ac6.png)

These are all the test builds I have when I checkout your fork.  I search for "test" and only the one named "test" comes up:

![image](https://user-images.githubusercontent.com/1209372/114950877-966c8f00-9e19-11eb-87f5-5db298d73949.png)

Reply:
```suggestion
		handle = NewFileSearch(main.buildPath..self.subPath..filterList.."*.xml")
```

I believe that's all that needs to change to fix this

Reply:
Thanks, but, i request you to check this case: 

**Filename:** "Test build necromancer volatile death.xml"
**Search:** "necro"  
**Result:** not match

You need to wrap the search term between '*' asterisks.

In my suggestion, i do not put it hardcoded in order to give the user the posibility to make more complex searchs.

Reply:
I suppose that this will be better:
`		handle = NewFileSearch(main.buildPath..self.subPath.."*"..filterList.."*.xml")`

Reply:
Added some commits to fix it. [This](https://github.com/PathOfBuildingCommunity/PathOfBuilding/pull/2546/commits/33cccbef9fa3dff8bee5b005ccb29b11a3977e22) for instance